### PR TITLE
Auto update frontend source in docker image and provide docker compose

### LINKED ISSUE #7: Expose frontend config as javascript file under `/rum-config.js`
To configure the embedded RUM agent, the agreed convention is to expose a Javascript file under `/rum-config.js`, which sets the agent config on the global `window` object, e.g.

    window.elasticApmJsBaseServerUrl = {{ ELASTIC_APM_JS_SERVER_URL }}

See https://github.com/elastic/opbeans-python/commit/5ee0562f0351, https://github.com/elastic/opbeans-go/commit/d0e527c763 for implementation examples.

### PR DESCRIPTION
**Pull request by alvarolobato**
Fixes elastic/opbeans-java#7 and elastic/opbeans-java#8

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
+version: "2.1"
```
Comment (felixbarny) [inline docker-compose.yml]: What is this file needed for? Don't we have `apm-integration-testing` to generate the `docker_compose.yml`?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    image: opbeans/opbeans-java:latest
+    ports:
+      - "127.0.0.1:${OPBEANS_SERVER_PORT:-8000}:8000"
+    logging:
+      driver: 'json-file'
+      options:
+          max-size: '2m'
+          max-file: '5'
+    environment:
+      - ELASTIC_APM_SERVICE_NAME=${ELASTIC_APM_SERVICE_NAME:-opbeans-java}
```
Comment (felixbarny) [inline docker-compose.yml]: Seems like `OPBEANS_JAVA_PROFILE` and maybe also`DATABASE_URL`, `DATABASE_DRIVER` and `DATABASE_DIALECT` are missing as you have removed the default values?

File: docker-compose.yml (hunk 1 above)
Comment (alvarolobato) [reply docker-compose.yml]: Other agents are providing a docker compose to just test this agent with an Elasticsearch, Kibana and Server for convenience. This is it.

See:
https://github.com/elastic/opbeans-go/blob/master/docker-compose.yml
https://github.com/elastic/opbeans-python/blob/master/docker-compose.yml

File: docker-compose.yml (hunk 2 above)
Comment (alvarolobato) [reply docker-compose.yml]: I've removed the default from the docker file, but not from the opbeans application: 
https://github.com/elastic/opbeans-java/blob/master/opbeans/src/main/resources/application.properties#L13-L16

I didn't like that the dockerfile was always overriding the application defaults and in this case for the docker-compose I wanted to use H2 and make it simple.
